addons/web_editor/views/snippets.xml

```diff
@@ -719,6 +731,19 @@
                 <we-button class="fa fa-fw fa-rotate-right" data-name="img_shape_transform_rotate_y_opt" data-set-img-shape-rotate-right="true"
                     data-no-preview="true" data-dependencies="shape_img_opt" title="Rotate right"/>
             </we-row>
+            <we-row t-if="not no_animations" string="Speed" class="o_we_sublevel_1">
+                <we-range data-dependencies="shape_img_opt" data-name="shape_anim_opt" t-att-string="animation_speed_label"
+                          t-att-class="indent and 'o_we_sublevel_2'"
```

Comment:
When you drag'n'drop a Cover snippet, the level of indentation is incremented. 
![image](https://github.com/odoo/odoo/assets/22252071/cdf60fab-8166-493a-8f21-80434503fd46)
Instead of a simple image shape : 
![image](https://github.com/odoo/odoo/assets/22252071/5cc9ed20-6f8a-429d-9458-f7182107410d)

Reply:
@SergeBayet My comment was about the image shape option not the background shape option where the "o_we_sublevel_2" class is also used but not in a `t-att-class` attribute...
The incremented indentation you show me in the cover snippet options is from the background shape option.

Reply:
Yep, in this case, it seems to be useless.